_schemaV2.graphql

```diff
@@ -46,8 +46,8 @@ type AccountRequest {
   notes: String
 }
 
-# An rail to show a list of user activity
-type ActivityRailHomeViewSection implements GenericHomeViewSection & Node {
+# An  to show a list of user activity
+type ActivityHomeViewSection implements GenericHomeViewSection & Node {
```

Comment:
#emphatically-suggested-but-non-blocking 😄 Since all of these types are very specific to the home view, and we want to be able to easily scan for homeview types in the type graph (as well as at a glance in code) the best way to name multiple variants of a category is to stick to a common prefix:

```
HomeViewActivitySection
HomeViewArticlesSection
HomeViewFairsSection
etc
```

And then we're not hunting all around graphiql and our schema for things that could be automatically grouped together. 

See the line numbers to get an idea of my point, which will start to show up at scale in our code and elsewhere (including file-names):

<img width="209" alt="Screenshot 2024-09-03 at 10 48 18 PM" src="https://github.com/user-attachments/assets/64e7aee7-cca9-4a95-9c5a-e7bfdc0c1f64">

Reply:
+ here, plus with the alphabetical sorting makes super easier to trace

Reply:
Thanks for the fix! Resolving.

Reply:
Made the change in https://github.com/artsy/metaphysics/pull/5962/commits/2c5513da77f6f57cdf0c61af2ef877882dc6be4a

Only difference is I went with e.g. `HomeViewSectionActivity` instead of `HomeViewActivitySection` — while the latter reads more naturally, the former actually does a better job with colocating and skimmability, which was the stated goal.

Reply:
totally 👌